src/test/mir-opt/nll/region-subtyping-basic.rs

```diff
@@ -19,23 +21,3 @@ fn main() {
         use_x(22);
     }
 }
-
-// END RUST SOURCE
-// START rustc.main.nll.0.mir
-// | '_#2r | U0 | {bb2[0..=8], bb3[0], bb5[0..=2]}
-// | '_#3r | U0 | {bb2[1..=8], bb3[0], bb5[0..=2]}
-// | '_#4r | U0 | {bb2[4..=8], bb3[0], bb5[0..=2]}
-// END rustc.main.nll.0.mir
-// START rustc.main.nll.0.mir
-// let _2: &'_#3r usize;
-// ...
-// debug p => _2;
-// ...
-// let _6: &'_#4r usize;
-// ...
-// debug q => _6;
-// ...
-// _2 = &'_#2r _1[_3];
-// ...
-// _6 = _2;
-// END rustc.main.nll.0.mir
```

Comment:
cc @nikomatsakis @matthewjasper are you okay with changing NLL tests to the new system that contains the entire MIR output?

Reply:
I'm ok with that. Honestly, we could delete those particular tests, they were useful in getting started but I'm not sure they provide much value now.

Reply:
I still think they're useful as canaries, to be able to detect when something changes.
Maybe useful for polonius?